.github/workflows/wheels.yml

```diff
@@ -85,34 +35,13 @@ jobs:
           - [ubuntu-22.04-arm, manylinux_aarch64, ""]
           - [ubuntu-22.04-arm, musllinux_aarch64, ""]
           - [macos-13, macosx_x86_64, openblas]
-          - [macos-13, macosx_x86_64, accelerate]
```

Comment:
Did you want to remove all macos-13 runs here?

Reply:
Can we do that separately based on what comes out of the discussion at gh-29728?

I haven't yet looked into what Azure plans to do with their platform support. Also, I hope we'll get rid of Azure completely ....

Reply:
For this one, my thinking was that there's another job that tests Accelerate LP64 and ILP64 at once (see [here](https://github.com/numpy/numpy/blob/83402f7a43363e8041b8f297fdd4f76a7fa9fea1/.github/workflows/macos.yml#L142-L151), plus this particular job is quite slow, so I wanted to remove this one as duplicate.